### PR TITLE
Add CosmosKit integration

### LINKED ISSUE #44: Add Cosmokit Integration
Switch custom wallet logic to use [CosmosKit ](https://cosmoskit.com/) instead. This will allow us to add support for new wallet very quickly.

- [x] Change custom wallet logic to use CosmosKit
- [x] Add support for old and new wallets
  - [x] Keplr
  - [x] Leap
  - [x] Keplr Mobile (NEW)
  - [x] Cosmostation (NEW)
  - [x] Cosmostation Mobile (NEW)
- [x] Sandbox Support
- [x] USDC Support

### PR DESCRIPTION
**Pull request by Redm4x**
- Switch to CosmosKit for wallet management
- Added support for Cosmostation + Cosmostation Mobile wallets
- Added support for Keplr Mobile wallet
- Added bundle-analyzer to track bundle size

### CODE REVIEW

File: deploy-web/src/context/CustomChainProvider/CustomChainProvider.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+          akash: { rest: [settings.apiEndpoint], rpc: [settings.rpcEndpoint] },
+          "akash-sandbox": { rest: [settings.apiEndpoint], rpc: [settings.rpcEndpoint] },
+          "akash-testnet": { rest: [settings.apiEndpoint], rpc: [settings.rpcEndpoint] }
+        }
+      }}
+      signerOptions={{
+        preferredSignType: chain => "direct",
+        signingStargate: chain => ({
+          registry: customRegistry,
+          gasPrice: GasPrice.fromString("0.025uakt")
```
Comment (baktun14) [inline deploy-web/src/context/CustomChainProvider/CustomChainProvider.tsx:38]: How does the gas work with cosmoskit?

Edit: Ok we still estimate the tx fees

File: deploy-web/src/styles/index.css [hunk 2]
```diff
@@ -4,6 +4,10 @@
   white-space: nowrap;
 }
 
+zag-portal > div {
+  z-index: 1500 !important;
+}
```
Comment (baktun14) [inline deploy-web/src/styles/index.css:9]: `zag-portal` is a custom html element?

**Review by baktun14 — APPROVED** [review]

File: deploy-web/src/styles/index.css (hunk 2 above)
Comment (Redm4x) [reply deploy-web/src/styles/index.css:9]: Yeah, cosmoskit puts the modals there with a default `z-index` of `999` which is lower than our header
image

Comment (baktun14) [reply deploy-web/src/styles/index.css:9]: Ok good to know.

**Comment by schnetzlerjoe** [comment]
In this commit (https://github.com/schnetzlerjoe/cloudmos/tree/features/cosmoskit-integration) I add one line of code to this PR to add Metamask support. We have been working the past week on adding Metamask to Cloudmos using the old wallet context. This adds it to the new context provider as well whenever you guys merge this!

**Comment by baktun14** [comment]
@schnetzlerjoe This is great thanks! cc @anilmurty

File: deploy-web/src/context/WalletProvider/WalletProvider.tsx [hunk 3]
```diff
@@ -161,18 +172,20 @@ export const WalletProvider = ({ children }) => {
...
+      //const simulation = await client.simulate(walletAddress, msgs, "");
+      // const estimatedFees = {
+      //   amount: [
+      //     {
+      //       amount: "0.025",
+      //       denom: uAktDenom
+      //     }
+      //   ],
+      //   gas: Math.ceil(simulation * 1.25).toString()
+      // };
```
Comment (baktun14) [inline deploy-web/src/context/WalletProvider/WalletProvider.tsx]: ✂️

**Review by baktun14 — APPROVED** [review]

File: deploy-web/src/context/WalletProvider/WalletProvider.tsx (hunk 3 above)
Comment (baktun14) [reply deploy-web/src/context/WalletProvider/WalletProvider.tsx]: Ah it's draft.

**Comment by Redm4x** [comment]
Metamask support will be added in a second step, progress will be tracked in #62

**Pull request merged**